lib/common/exceptions/service_outage.rb

```diff
@@ -13,7 +13,7 @@ def initialize(outage = nil, options = {})
       end
 
       def errors
-        Array(SerializableError.new(i18n_data.merge(detail: @detail)))
+        Array(SerializableError.new(i18n_data.merge(detail: @detail, status: 503)))
```

Comment:
The [ExceptionHandling](https://github.com/department-of-veterans-affairs/vets-api/blob/710119ac09e28f9bfe43fed7467711c499afa094/app/controllers/concerns/exception_handling.rb) concern that's included in [ApplicationController](https://github.com/department-of-veterans-affairs/vets-api/blob/f9230a47d7887424b0b16c4de3c62f9ce2b58956/app/controllers/application_controller.rb) includes a [handle_errors](https://github.com/department-of-veterans-affairs/vets-api/blob/710119ac09e28f9bfe43fed7467711c499afa094/app/controllers/concerns/exception_handling.rb#L62-L69) function which returns JSON to the client.  It needs the exception to have a `status_code` to pass along to the client.

`ServiceOutage` inherits from [BaseError](https://github.com/department-of-veterans-affairs/vets-api/blob/455c9b9370a36e5490bb1c2176a865a1024245c0/lib/common/exceptions/base_error.rb#L11-L16) which sets the `status_code` as the value of the `status` in the first error